docs/usage/self-hosted-configuration.md

```diff
@@ -7,6 +7,10 @@ description: Self-Hosted Configuration usable in renovate.json or package.json
 
 The below configuration options are applicable only if you are running your own instance ("bot") of Renovate.
 
+## allowedPostUpgradeTasks
```

Comment:
Let's add some examples to this description

Reply:
I have added two examples